ts-packages/web/src/app/(social)/_components/create-repost.tsx

```
@@ -474,8 +504,74 @@ export const RePostDraftProvider: React.FC<{ children: React.ReactNode }> = ({
+      );
+    }
+  }, [content, maxLines, minLength]);
+
+  return (
+    <div className={`${className} relative`} ref={containerRef}>
+      <div
+        ref={contentRef}
+        className={`${contentClassName} ${
+          !isExpanded ? `line-clamp-${maxLines} overflow-hidden` : ''
+        }`}
+        dangerouslySetInnerHTML={{
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)